scripts/g.bands/landsat.json

```diff
@@ -0,0 +1,41 @@
+{
+    "Landsat7": {
```

Comment:
TODO: also add L8 to the file

Reply:
L8 added in 479af79dae.

```
> g.bands band=L8
L8_1
L8_2
L8_3
L8_4
L8_5
L8_6
L8_7
L8_8
L8_9
L8_10
L8_11

> g.bands band=L8 -e
description: The Landsat 8 bands
shortcut: L8
instruments: Operational Land Imager (OLI) and the Thermal Infrared Sensor (TIRS)
launched: February 11, 2013
source: https://www.usgs.gov/land-resources/nli/landsat/landsat-8
    band: 1
        bandwidth (µm): 0.43-0.45
        resolution (meters): 30
        tag: Visible
    band: 2
        bandwidth (µm): 0.45-0.51
        resolution (meters): 30
        tag: Visible (Blue)
    band: 3
        bandwidth (µm): 0.53-0.59
        resolution (meters): 30
        tag: Visible (Green)
    band: 4
        bandwidth (µm): 0.64-0.67
        resolution (meters): 30
        tag: Visible (Red)
    band: 5
        bandwidth (µm): 0.85-0.88
        resolution (meters): 30
        tag: Near-Infrared
...
```

I also added a new attribute `tag`. @veroandreo @neteler Any idea about better name?

Reply:
> > will this new feature interact with temporal algebra, e.g., how would you estimate NDVI from a STRDS which is an image collection? Using the filtering notation within the expression? Since this is an example I've seen in openEO project, would be great to understand h
> 
> @veroandreo Very good question! I was thinking about syntax below.
> 
> ```
> > t.rast.mapcalc input=test output=ndvi expression="float(test:S2B_8 - test:S2B_4) / ( test:S2B_8 + test:S2B_4 )" basename=ndvi
> ```
Take into account that in `t.rast.algebra` the ":" is used for temporal selection. See https://grass.osgeo.org/grass76/manuals/t.rast.algebra.html#temporal-selection